src/mpc-hc/MainFrm.cpp

```diff
@@ -5203,7 +5203,7 @@ void CMainFrame::OnFileSubtitlesLoad()
         dwFlags |= OFN_DONTADDTORECENT;
     }
     CString filters;
-    filters.Format(_T("%s|*.srt;*.sub;*.ssa;*.ass;*.smi;*.psb;*.txt;*.idx;*.usf;*.xss;*.rt;*.sup|%s"),
+    filters.Format(_T("%s|*.srt;*.sub;*.ssa;*.ass;*.smi;*.psb;*.txt;*.idx;*.usf;*.xss;*.rt;*.sup|%s;*.vtt"),

```

Comment:
I think it should be:
*.sup;*.vtt|%s

Reply:
Agreed. I can submit or you can just fix it.